lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -6,10 +6,16 @@
                 .replace(/>/g, '&gt;')
         }
     %>
-    <div class='exam-text'>
+    <div class='exam-text js-exam-text' data-show-long="true">
         <% // xss-lint: disable=underscore-not-escaped %>
-        <%= interpolate_text('You are taking "{exam_link}" as a {exam_type} exam. The timer on the right shows the time remaining in the exam.', {exam_link: "<a href='" + exam_url_path + "'>"+gtLtEscape(exam_display_name)+"</a>", exam_type: (!_.isUndefined(arguments[0].exam_type)) ? exam_type : gettext('timed')}) %>
-        <%- gettext('To receive credit for problems, you must select "Submit" for each problem before you select "End My Exam".') %>
+        <%= interpolate_text('You are taking "{exam_link}" as a {exam_type} exam. ', {exam_link: "<a href='" + exam_url_path + "'>"+gtLtEscape(exam_display_name)+"</a>", exam_type: (!_.isUndefined(arguments[0].exam_type)) ? exam_type : gettext('timed')}) %>
+        <span class="js-exam-additional-text" aria-hidden="false">
+            <%- gettext('The timer on the right shows the time remaining in the exam.') %>
+            <%- gettext('To receive credit for problems, you must select "Submit" for each problem before you select "End My Exam".') %>
+        </span>
+        <button class="js-toggle-show-more toggle-show-more btn btn-pl-link btn-link" data-show-more-text="<%- gettext('Show More') %>" data-show-less-text="<%- gettext('Show Less') %>">
```

Comment:
Is it bad practice to leverage a single class for both styling and for use with Javascript (i.e. the need for both `js-toggle-show-more` and `toggle-show-more`). I am wondering why you can't use `toggle-show-more` for both?

Reply:
I follow that line of thinking, yes. It's the single responsibility principle, and makes it easier to investigate questions like "is this class still necessary?"